src/shell/components/global-notifications/GlobalNotifications.js

```diff
@@ -72,95 +133,155 @@ export default connect((state) => {
+
+          <Drawer
+            className={styles.Drawer}
+            position="right"
+            offset="0px"
+            width="450px"
+            height="calc(100vh - 40px)"
+            open={drawerOpen}
           >
-            <FontAwesomeIcon
-              className={cx(
-                props.notifications[0].kind
-                  ? styles[props.notifications[0].kind]
-                  : null,
-                styles.Icon
+            <DrawerContent className={styles.DrawerContent}>
+              <header>
+                <h1 className={styles.display}>My Notifications</h1>
+                <AppLink to="/reports/audit-trail">View All Logs</AppLink>
+              </header>
+
+              {!props.notifications.length && (
+                <h2 className={styles.headline}>No actions taken</h2>
               )}
-              icon={msgIcon(props.notifications[0].kind)}
-            />
-            <p className={styles.Message}>{props.notifications[0].message}</p>
-          </div>
-        )}
-
-        <Drawer
-          className={styles.Drawer}
-          position="right"
-          offset="0px"
-          width="450px"
-          height="calc(100vh - 40px)"
-          open={drawerOpen}
-        >
-          <DrawerContent className={styles.DrawerContent}>
-            <header>
-              <h1 className={styles.display}>My Notifications</h1>
-              <AppLink to="/reports/audit-trail">View All Logs</AppLink>
-            </header>
-
-            {!props.notifications.length && (
-              <h2 className={styles.headline}>No actions taken</h2>
-            )}
-
-            <ul>
-              {props.notifications.map((notice, i) => {
-                return (
-                  <li
-                    key={i}
-                    className={cx(styles.Notification, styles.bodyText)}
-                  >
-                    <p className={styles.Message}>
-                      <FontAwesomeIcon
-                        className={cx(
-                          notice.kind ? styles[notice.kind] : null,
-                          styles.Icon
-                        )}
-                        icon={msgIcon(notice.kind)}
-                      />
-                      {notice.message}
-                    </p>
-                    {notice.epoch && (
-                      <small className={cx(styles.Timestamp, styles.caption)}>
-                        {moment(notice.epoch).fromNow()}
-                      </small>
-                    )}
-                  </li>
-                );
-              })}
-            </ul>
-          </DrawerContent>
-        </Drawer>
-      </aside>
+
+              <ul>
+                {props.notifications.map((notice, i) => {
+                  return (
+                    <li
+                      key={i}
+                      className={cx(styles.Notification, styles.bodyText)}
+                    >
+                      <p className={styles.Message}>
```

Comment:
Wouldn't it be better to convert these elements to MUI components instead?

Reply:
If I update this to MUI Components it will update also the designs cause its using old design. If that won't be an issue I can proceed to update it?